test/csi/simple/Makefile

```diff
@@ -1,16 +1,22 @@
 TARGETS := fib-serial foo memops
```

Comment:
Do you think there's a way we could perhaps specify a tool using a flag? I'm not particularly strong with Makefiles, but it'd be really nice to be able to run, e.g., `make TOOL=print_tool`, and have it compile the tests with the print tool. If not, no worries; we can just modify the Makefile as we're testing.